documentation/asciidoc/computers/raspberry-pi/usb-bus-on-raspberry-pi.adoc

```diff
@@ -4,7 +4,7 @@ In general, every device supported by Linux can be used with a Raspberry Pi, alt
 
 === Maximum Power Output
 
-As with all computers, the USB ports on the Raspberry Pi supply a limited amount of power. Often problems with USB devices are caused by power issues. To rule out power as the cause of an problem, connect your USB devices to the Raspberry Pi using a powered hub.
+As with all computers, the USB ports on the Raspberry Pi supply a limited amount of power. Often problems with USB devices are caused by power issues. To rule out power as the cause of a problem, connect your USB devices to the Raspberry Pi using a powered hub.
```

Comment:
Should be "the problem" not "a" or "an".

Reply:
I guess you could also use "To rule out insufficient power as..." ?

Reply:
Yup, that would be better.

Reply:
@lurch @aallan 
Thanks advice.